atomic_reactor/schemas/config.json

```diff
@@ -122,6 +122,58 @@
         "additionalProperties": false,
         "required": ["hub_url", "root_url", "auth"]
     },
+    "remote_hosts": {
+        "description": "Remote-host instance",
+        "type": "object",
+        "properties": {
+            "slots_dir": {
+                "description": "Remote-host slots directory",
+                "type": "string"
+            },
+            "pools": {
+                "description": "Pool of Remote-hosts",
+                "type": "object",
+                "patternProperties": {
+                    "^[a-zA-Z0-9_]*$": {
+                        "description": "Remote host platforms",
+                        "type": "object",
+                        "patternProperties": {
+                            "^[a-zA-Z0-9_.-]*$": {
```

Comment:
Should be `+` as well?

Reply:
Do you mean replace `*` with `+` to expect at least 1 character here? I can do that ... Because I don't expect the `plus` symbol to appear in the host name.

Reply:
well * or + doesn't matter, it can't be empty .....

he meant + in the hostname which is unlikely

Reply:
ok I'll leave it